Lib/_compression.py

```diff
@@ -110,6 +110,16 @@ def read(self, size=-1):
         self._pos += len(data)
         return data
 
+    def readall(self):
+        chunks = []
+        # sys.maxsize means the max length of output buffer is unlimited,
+        # so that the whole input buffer can be decompressed within one
+        # .decompress() call.
+        while data := self.read(sys.maxsize):
```

Comment:
maybe we don't support this operator yet

Reply:
I did see a reference on [L84](https://github.com/RustPython/RustPython/blob/main/compiler/ast/src/unparse.rs#L84) of `compiler/ast/src/unparse.rs` but it might not be handling it as the assignment operator described in [PEP 572](https://peps.python.org/pep-0572/).